_pages/events.md

```diff
@@ -5,24 +5,30 @@ image: images/gen.jpg
+### 32c3 (December, 2015)
+At [Chaos Computer Club](https://events.ccc.de/category/32c3/) (#32C3), Jennifer Helsby, CTO of Lucy Parsons Labs, presented on the mass quantities of data being incorporated into predictive systems. While these algorithms operate in the dark their use has had many consequences, both intentional and unintentional. Her talk covered issues of fairness and accountability involved in controlling algorithms for media, policy, and policing.
 
-## Community Meetings  
+### DEF CON 23 (August, 2015)
+At [DEF CON 23](https://www.defcon.org/), Jennifer Helsby presented in the Crypto and Privacy Village on the privacy implications of machine learning. Freddy Martinez presented on IMSI Catchers and practical counter-surveillance. ([Video](https://www.youtube.com/embed/JyTb5mJOYLo))
 
-[Chi Hack Night](https://chihacknight.org/) is Chicago’s weekly event to build, share and learn about civic tech. In June of 2016, Jennifer Helsby and Freddy Martinez presented on the recent deployment of a SecureDrop system hosted by Lucy Parsons Labs. View video. The presentation was recorded and can be seen [here](https://www.youtube.com/watch?v=XKY_Sgv9ej8).
+### THOTCON 0x5 (April, 2014)
+At [THOTCON 0x5](http://thotcon.org/), a Chicago-area information security conference, Freddy Martinez discussed the technical workings of IMSI Catchers and his work to uncover their use by police in Chicago.
 
-The [Electronic Frontier Alliance](https://www.eff.org/electronic-frontier-alliance) (EFA) hosted a workshop on surveillance self-defense in Chicago in March, 2016. Jennifer Helsby presented on the current state of local police surveillance. Groups from across the city participated, including [Chicago GNU/Linux Users Group](https://chicagolug.org/), [Restore the Fourth-Chicago](http://rt4chicago.com/index.html), and the National Lawyers Guild [Chicago Chapter](http://nlgchicago.org/). [Learn more](https://www.eff.org/deeplinks/2016/04/community-groups-come-together-across-us-promote-digital-rights). 
+## Community Meetings
 
-In partnership with Chicago [CryptoParty](https://www.cryptoparty.in/chicago), Lucy Parsons Labs organizes events designed to pass on knowledge about how to protect yourself in the digital space. This can include encrypted communication, preventing being tracked while browsing the web and general security advice for computers and smartphones. In these hands-on workshops, participants are encourage to bring their devices and instal security applications. 
+[Chi Hack Night](https://chihacknight.org/) is Chicago’s weekly event to build, share, and learn about civic tech. In June of 2016, Jennifer Helsby and Freddy Martinez presented on the recent deployment of a SecureDrop system hosted by Lucy Parsons Labs. ([Video](https://www.youtube.com/watch?v=XKY_Sgv9ej8))
```

Comment:
"recent" is out of date. also maybe we add a link to the SD page/

Reply:
Pushed